index.js

```diff
@@ -19,6 +31,26 @@ const manageRecipes = async () => {
     await Recipe.deleteMany();
 
     // Run your code here, after you have insured that the connection was made
+    
+    
+    await Recipe.create(newRecipe);
+
+    
+    let allRecipes = await Recipe.insertMany(data);
+    console.log(allRecipes);
```

Comment:
If you wanted to `console.log` only the titles of the recipes, you needed to loop over your `allRecipes` variable that is storing an array with your recipes objects.
Something like this:
```
// console log to see the titles of the inserted recipes
    for (let i = 0; i < allRecipes.length; i++) {
      console.log(allRecipes[i].title);
    }
```